### PR TITLE
Enterprise Bot: Updates for Localization and Channel Support

### PR DESCRIPTION
**Pull request by lauren-mills**
## Description

- Added deployment folder for each supported language (en, de, fr, it, es, zh)
- Added localized responses for all supported languages 
- Added support for speech responses in template manager files
- Added Extension to check for starting message for primary channels (Emulator, Skype, Teams, Cortana, WebChat)
- Updated to latest packages

## Testing Steps

Once this is merged in, I will start a build so we can all test from the myget feed.

### CODE REVIEW

File: templates/Enterprise-Template/src/csharp/EnterpriseBotTemplate/Bot Framework/Extensions/ActivityExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                        {
+                            return true;
+                        }
+
+                        return false;
+                    }
+
+                default:
+                    {
+                        if (activity.Type == ActivityTypes.ConversationUpdate)
```
Comment (ryanisgrig) [inline templates/Enterprise-Template/src/csharp/EnterpriseBotTemplate/Bot Framework/Extensions/ActivityExtensions.cs:40]: So MSTeams falls under this case. Can you explain how the old check (`if (!activity.MembersAdded.Any(m => m.Id == activity.Recipient.Id))`) failed for Teams, is it the User Id that's different from the emulator?

**Comment by ryanisgrig** [comment]
Resolves #280

File: templates/Enterprise-Template/src/csharp/EnterpriseBotTemplate/Bot Framework/EnterpriseBot.cs [hunk 2]
```diff
@@ -45,36 +44,27 @@ public class $safeprojectname$ : IBot
...
-            if (result.Status == DialogTurnStatus.Empty)
+            if (dc.ActiveDialog != null)
+            {
+                var result = await dc.ContinueDialogAsync();
+            }
+            else
             {
-                // Handle Cortana's launch action
-                if (CortanaHelper.IsLaunchActivity(turnContext.Activity))
+                if (turnContext.Activity.IsStartActivity())
```
Comment (ryanisgrig) [inline templates/Enterprise-Template/src/csharp/EnterpriseBotTemplate/Bot Framework/EnterpriseBot.cs:64]: Will we have to update each skill with this?

Comment (ryanisgrig) [reply templates/Enterprise-Template/src/csharp/EnterpriseBotTemplate/Bot Framework/EnterpriseBot.cs:64]: Synced offline

**Review by ryanisgrig — APPROVED** [review]

File: templates/Enterprise-Template/src/csharp/EnterpriseBotTemplate/Bot Framework/Extensions/ActivityExtensions.cs (hunk 1 above)
Comment (lauren-mills) [reply templates/Enterprise-Template/src/csharp/EnterpriseBotTemplate/Bot Framework/Extensions/ActivityExtensions.cs:40]: Teams, like webchat, requires that the user sends a message first to start the conversation. With the other changes, everything works as expected in Teams now.

**Pull request merged**